java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -258,7 +258,9 @@
 import glide.api.models.commands.function.FunctionRestorePolicy;
 import glide.api.models.commands.geospatial.GeoAddOptions;
 import glide.api.models.commands.geospatial.GeoSearchOptions;
-import glide.api.models.commands.geospatial.GeoSearchOrigin;
+import glide.api.models.commands.geospatial.GeoSearchOrigin.CoordOrigin;
+import glide.api.models.commands.geospatial.GeoSearchOrigin.MemberOrigin;
```

Comment:
Looks like these only appear in the docs/examples, do we need to import these?

Reply:
Yes, otherwise doc has broken links.